README.md

```diff
@@ -78,10 +80,10 @@ react-native run-ios
   * in case of handler's failure, `failure` prop (if defined) will be run.
   * Combining `onEnter`, `onExit`, `success`, and `failure` makes patterns like authentication, data validation, and conditional transitions simple and intuitive.
 * [MobX](https://mobx.js.org/)-friendly: all scenes are wrapped with `observer`. You may subscribe to `navigationStore` (`Actions` in v3) and observe current navigation state. Not applicable to Redux.
-* Flexible nav bar customization not currently allowed by React Navigation:
+* Flexible Nav bar customization ,currently not allowed by React Navigation:
```

Comment:
Flexible Nav bar customization, currently not allowed by React Navigation:

Reply:
I see comma right before currently, so now it is "Flexible Nav bar customization ,currently not allowed by React Navigation:"